translations/ui-finc-config/en_US.json

```diff
@@ -1,63 +1,77 @@
 {
-    "meta.title": "Finc Config",
```

Comment:
I'm not sure if devs are required to touch en_US.json, @dltj might provide a valuable input on this